lms/templates/financial-assistance/financial_assessment_form.underscore

```diff
@@ -36,12 +36,12 @@
 		</div>
 	</div>
 
-	<%= fields %>
+	<%= HtmlUtils.HTML(fieldsHtml) %>
```

Comment:
@andy-armstrong: `ensureHtml()` is safer and the better choice because it ensures the fieldsHtml is properly escaped, whether it is an HtmlSnippet or not.  However, `HTML()` shouldn't double-escape, because it is a way for you to simply declare something as containing HTML, which would hopefully be used only when you know you have safe HTML that was HTML before HtmlUtils could have gotten involved.